conda/meta.yaml

```diff
@@ -22,18 +22,22 @@ test:
     - pyproject.toml
   commands:
     - rs_download_data    
-    - py.test -vs
+    - pytest -vs
 
 requirements:
   host:
-    - python
+    - python {{ python }}
     - pip
     - setuptools_scm
     - setuptools
-    - pytest-runner
+  build:
+    - python {{ python }}
+    - setuptools_scm
+    - setuptools
+    - ts-conda-build =0.3
```

Comment:
Isn't `- rubin-sim >=1` required in this `build1` section as well?

Reply:
No, not really.. Here's the doc about that section: https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#build